webpack.config.shared.js

```diff
@@ -26,15 +26,13 @@ module.exports = {
   },
   babelSharedLoader: {
     test:    /\.jsx?$/,
-    exclude: /node_modules/,
```

Comment:
Can we do something like [this](https://github.com/webpack/webpack/issues/2031#issuecomment-283517150) and specifically include certain node_modules files instead of including the full contents?

Reply:
What files or directories would be included though? As far as I can tell it is complaining about arrow functions being referenced which could be used in any number of dependencies or libraries they depend on, etc.

Reply:
Ah right, sorry I misunderstood the root problem